src/Application/Itmo.Dev.Asap.Github.Application/CommandExecution/PullRequestContextCommandVisitor.cs

```diff
@@ -34,8 +35,18 @@ public class PullRequestContextCommandVisitor : ISubmissionCommandVisitor
 
     public async Task<SubmissionCommandResult> VisitAsync(ActivateCommand command)
     {
-        GithubUser issuer = await _context.Users.GetForGithubIdAsync(_pullRequest.SenderId);
-        GithubSubmission submission = await _context.Submissions.GetSubmissionForPullRequestAsync(_pullRequest);
+        GithubUser? issuer = await _context.Users.FindByGithubIdAsync(_pullRequest.SenderId);
+        if (issuer is null)
+        {
+            return new SubmissionCommandResult.Failure(ActivateErrorMessage.IssuerNotFound());
+        }
+
+        GithubSubmission? submission = await _context.Submissions.FindSubmissionForPullRequestAsync(_pullRequest);
+
+        if (submission is null)
+        {
+            return new SubmissionCommandResult.Failure(ActivateErrorMessage.SubmissionNotFound());
+        }
```

Comment:
у таких однострочных return'ов лучше скобки опускать, если возможно, чтобы код не раздувать 

```suggestion
        GithubUser? issuer = await _context.Users.FindByGithubIdAsync(_pullRequest.SenderId);

        if (issuer is null)
            return new SubmissionCommandResult.Failure(ActivateErrorMessage.IssuerNotFound());

        GithubSubmission? submission = await _context.Submissions.FindSubmissionForPullRequestAsync(_pullRequest);

        if (submission is null)
            return new SubmissionCommandResult.Failure(ActivateErrorMessage.SubmissionNotFound());
```